### ISSUE TITLE
Categories for applications

### ISSUE THREAD

**Issue by samcro1967** [pawelmalak/flame#142]
Is there a way to create categories for apps the same way you can for web link?

**Comment by gitcatpeter** [comment]
Mr developer has been rejecting this most requested feature for months lol

However I don't get why he does not like it.
Seems pretty logical to me to have way to partition apps, and could be
turned on/off in settings.
Why oh why?

**Comment by samcro1967** [comment]
Ah, thanks.  Guess I should have searched the issue before I opened this one then LOL.  I will stick with Plugsy then for now.  Like the docker labels approach and not having to modify a confg file, but app categories is worth it for only having to manage a few settings in a config file.

**Comment by mark4409** [comment]
This is something I'd like too. When I used to use the Homer dashboard, I had a separate section for all of the backend admin interfaces... eg postfixadmin, phpmyadmin, wordpress admin, funkwhale backend admin, etc etc. It does seem a little cluttered having everything together in a big list. The justification for having categorised applications is exactly the same as for having categorised bookmarks. I'm considering working around it for the moment by simply using bookmarks for everything - which probably goes against what you actually designed the site to do.

**Comment by mark4409** [comment]
Actually.... you could probably get away with not having application categories if there was a bit more flexibility in the titles. I'm just looking at the layout here and I'm thinking... if I could rename applications to 'Main Applications' and then rename bookmarks to 'Other', I could just use the bookmark categories for the rest ie My Websites, Backend Admin, External Services, Documentation, etc. Anything more could be easily tweaked in the custom CSS section.

**Comment by samcro1967** [comment]
I would agree bookmarks could probably be used by extending the code to accept two more docker labels.
     - flame.type=bookmark
     - flame.bookmark.category=Media

**Comment by holocronology** [comment]
Was just about to request this. +1

**Comment by syndr** [comment]
This is something I'd really like as well. I'd like to add my applications under the "applications" type, and actual bookmarks under "bookmarks", but instead I end up disabling applications entirely and putting everything under bookmarks, as I really value the categories.

**Comment by holocronology** [comment]
>  instead I end up disabling applications entirely and putting everything under bookmarks

That's a great idea.

**Comment by samcro1967** [comment]
You might wan to check out [Plugsy](https://github.com/plugsy/core).  It has categories for apps and works with Docker labels.  It does have one config file to maintain, but it is minimal.
